src/templates/test/scenarios/FunctionParametersScenario.mustache

```diff
@@ -65,7 +65,7 @@ contract {{ name.Plural }}FunctionParametersScenario {
             _{{ ../name.lowerCamelCase }}{{#if ../slotsArray}}[{{ @index }}]{{/if}} = _{{ ../name.lowerCamelCase }}DataSlot{{ @index }}[i];
             {{/each}}
             {{#each variables}}
-            unknownLenght{{ name.UpperCamelCase }}[i] = {{ ../name.Plural }}Coder.decode{{ ../name.UpperCamelCase }}{{ name.UpperCamelCase }}(_{{ ../name.lowerCamelCase }});
+            unknownLenght{{ name.UpperCamelCase }}[i] = decode{{ ../name.UpperCamelCase }}{{ name.UpperCamelCase }}(_{{ ../name.lowerCamelCase }});
```

Comment:
🙈

Reply:
it's just a name, come up with a better one and I'll approve your PR